### PR TITLE
Add documentation for {block,allow}_new_chunks

### PR DESCRIPTION
**Pull request by WireBaron**
This adds descriptions of the new allow_new_chunks and
block_new_chunks commands to api.md

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -376,6 +413,50 @@ SELECT attach_tablespace('disk2', 'conditions', if_not_attached => true);
...
+---
+## block_new_chunks() [](block_new_chunks)
+
+Prevent new chunks from being created on a data node without losing
+access to any of the existing data. Note than any chunks already
+existing on the data node may continue to receive further INSERTs and
+UPDATEs.
+
+This can be useful if one data node is running low on space or simply
+as a tool to fine tune how data is distributed as more distributed
```
Comment (solugebefola) [inline api.md]:
```suggestion
as a tool to fine-tune how data is distributed as more distributed
```

File: api.md [hunk 2]
```diff
@@ -376,6 +413,50 @@ SELECT attach_tablespace('disk2', 'conditions', if_not_attached => true);
...
+
+| Name             | Description                       |
+|------------------|-----------------------------------|
+| `data_node_name` | Name of data node to stop creating chunks for the distributed hypertable |
+
+#### Optional Arguments [](block_new_chunks-optional-arguments)
+
+| Name         | Description                            |
+|--------------|----------------------------------------|
+| `hypertable` | Name of the distributed hypertable for which chunks creation is being stopped. If NULL, no new chunks will be created for any hypertables. |
```
Comment (solugebefola) [inline api.md]:
```suggestion
| `hypertable` | Name of the distributed hypertable for which chunk creation is being blocked. If NULL, no new chunks will be created for any hypertables. |
```

File: api.md [hunk 3]
```diff
@@ -376,6 +413,50 @@ SELECT attach_tablespace('disk2', 'conditions', if_not_attached => true);
...
+
+#### Errors
+
+Blocking chunks on a data node is not permitted:
+- If it would result in no available data nodes for any future chunks
+on a hypertable
+- If it would result in future chunks being under-replicated for the
+distributed hypertable (without the `force` argument)
+
+>:TIP: Replication is currently experimental, and not a supported feature
```
Comment (erimatnor) [inline api.md]:
```suggestion
>:TIP: Replication is currently experimental, and not a supported feature.
```

File: api.md [hunk 4]
```diff
@@ -376,6 +413,50 @@ SELECT attach_tablespace('disk2', 'conditions', if_not_attached => true);
...
+## block_new_chunks() [](block_new_chunks)
+
+Prevent new chunks from being created on a data node without losing
+access to any of the existing data. Note than any chunks already
+existing on the data node may continue to receive further INSERTs and
+UPDATEs.
+
+This can be useful if one data node is running low on space or simply
+as a tool to fine-tune how data is distributed as more distributed
+hypertables are created. To resume using a data node once blocked for
```
[the remark this answers is not in the document]
Comment (WireBaron) [reply api.md]: I didn't see a good way of adding this without being exceptionally verbose.  I did change the first sentence to highlight this as a tool for safe destaging though.

**Pull request merged**